Dynatrace/dynatrace-operator
# Make CSI log GC more error resilient

# Description

Currently our CSI Log Garbage Collector stops, if there is an error when accessing the run folders. Meaning he does not collect a single directory, but returns and stops looping over the directories. We should prevent that and at least collect all the other directories which are fine.

## How can this be tested?

1. Create DynaKube with csiDriver usage
2. Create a faulty run folder (which has a mount to a location which does not exist)
3. Info log should appear, but other folders, which are not needed anymore should be garbage collected


## Checklist
